<?xml version="1.0"?>
<launch>

  <!-- Mux between joystick and commands -->
  <node name="cmd_vel_mux" pkg="topic_tools" type="mux" respawn="true"
        args="base_controller/command /cmd_vel /teleop/cmd_vel">
    <remap from="mux" to="cmd_vel_mux" />
  </node>

  <!-- Teleop -->
  <node name="joy" pkg="joy" type="joy_node" >
    <param name="autorepeat_rate" value="5"/>
  </node>
  <node name="teleop" pkg="etherbotix_python" type="simple_teleop.py" >
    <param name="axis_linear" value="4"/>
    <param name="axis_angular" value="0"/>
    <param name="scale_linear" value="1.0" />
    <param name="scale_angular" value="3.0" />
    <param name="axis_deadman" value="4"/><!--L1 shoulder button-->
  </node>

</launch>
